src/server/pkg/metrics/segment.go

```diff
@@ -23,6 +24,28 @@ func reportSegment(metrics *Metrics) {
 		},
 	})
 	if err != nil {
-		lion.Errorf("error reporting to Segment: %s", err.Error())
+		lion.Errorf("error reporting cluster metrics to Segment: %s", err.Error())
+	}
+}
+
+func identifyNewUser(userID string) {
+	err := client.Identify(&analytics.Identify{
+		UserId: userID,
+	})
+	if err != nil {
+		lion.Errorf("error reporting user identity to Segment: %s", err.Error())
+	}
+}
+
+func reportUserMetricsToSegment(allUsersActions countableUserActions) {
```

Comment:
Why have this be a separate function? It doesn't seem like there's much need for reporting 1 set of metrics and not the other.